### ISSUE TITLE
Automatic platform detection

### ISSUE THREAD

**Issue by pavolloffay** [wasmerio/wasmer-java#53]
### Motivation

Instead of including multiple jars (https://github.com/wasmerio/wasmer-java#development) I would like to include a single JAR that works on multiple platforms and have a single API to use the wasner.

### Proposed solution

Automatically detect platform.

### Alternatives

None.

### Additional context

None.

**Comment by Hywan** [comment]
Yup, that's something I will think about. You aren't the first one to ask that.

**Comment by zacharywhitley** [comment]
You might be interested in this https://github.com/scijava/native-lib-loader
